### DISCUSSION TITLE
Add-on idea: Connection to free and open-source helper and shift planning, such as engelsystem.de

### DISCUSSION THREAD

**Discussion by linux-lukas** [pretalx/pretalx#1189] [Ideas] [answered]
I could imagine that pretalx could very well be complemented by an optionally activate option, in the form of an add-on, with which the shift schedule and helper system could be connected.

Specifically, the idea would be the following: 

You create rooms and events in pretalx, and then you can assign helpers for these events via the shift plan system. So that there are no double event entries (in pretalx and something like the engelsystem.de).

What do you think @rixx ?

**Comment by rixx** [comment, marked as answer]
What would a new plugin add over the current integration with Engelsystem?

**Reply by linux-lukas** [reply]
Is the cooperation of pretalx and engelsystem documented somewhere? Is this realized by a plugin? I have not yet found anything in the web interface to work with in Engelsystem. @rixx

**Reply by rixx** [reply]
Engelsystem integrates with both pretalx and frab by importing the schedule.xcal export format.
